dll/win32/framedyn/CMakeLists.txt

```
@@ -14,7 +14,7 @@ set_module_type(framedyn win32dll UNICODE)
 add_importlibs(framedyn oleaut32 msvcrt kernel32 ntdll)
 add_cd_file(TARGET framedyn DESTINATION reactos/system32/wbem FOR all)
 
-if(NOT MSVC)
+if((NOT MSVC) AND (ARCH STREQUAL "i386"))
     add_asm_files(framedynex_asm alias_gcc.s)
     add_library(framedynex ${framedynex_asm})
     set_target_properties(framedynex PROPERTIES LINKER_LANGUAGE ASM)
```

I think this will be more correct, as this assembler file should probably be used for both x86 and amd64 architectures:
```suggestion
if(NOT MSVC)
    if (ARCH STREQUAL "i386" OR ARCH STREQUAL "amd64")
        add_asm_files(framedynex_asm alias_gcc.s)
        add_library(framedynex ${framedynex_asm})
        set_target_properties(framedynex PROPERTIES LINKER_LANGUAGE ASM)
    endif()
```
(would be nice if somebody else could confirm it)

---

The main target of this file is to create mangled names, so a correct(er) approach would be to add an arm-version of this as well. (Possibly just change the jmp in the existing file?)